src/App.js

```diff
@@ -3,11 +3,57 @@ import './App.css'
 
 
 class App extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      color: '#000000',
+      usedColors: []
+    }
+
+    this.changeColor = this.changeColor.bind(this)
+  }
+
+  getNewColor() {
+    return '#' + Math.floor(Math.random()*16777215).toString(16)
+  }
+
+  changeColor() {
```

Comment:
What is the runtime of this function and is there any way to improve the runtime? If so, please update